### PR TITLE
[copy update] /download/amd-xilinx

### PR DESCRIPTION
**Pull request by mtruj013**
## Done

- Updated copy and added new tab as per [doc](https://docs.google.com/document/d/1cxQnEErs8FZ3dZ_IhlfcaJ03gE_vrMJfRC-5me9IDd4/edit#)
- Drive by's:
  - Divided tabs into separate files so that they're easier to maintain 

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: https://ubuntu-com-12902.demos.haus/download/amd-xilinx
    - Be sure to test on mobile, tablet and desktop screen sizes
- See that the requested changes have been made

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-3924

### CODE REVIEW

File: templates/download/amd-xilinx/index.html [hunk 1]
```diff
@@ -2,19 +2,19 @@
...
+{% block meta_copydoc%} https://docs.google.com/document/d/1cxQnEErs8FZ3dZ_IhlfcaJ03gE_vrMJfRC-5me9IDd4 {% endblock meta_copydoc %}
 
 {% block content %}
 
 <section class="p-strip--suru-topped">
   <div class="row u-vertically-center">
     <div class="col-6">
-      <h1>Install Ubuntu on Xilinx</h1>
-      <p>Run Ubuntu on your Xilinx Zynq UltraScale+ MPSoC-based evaluation boards and Kria SOMs. </p>
+      <h1>Install Ubuntu on AMD-Xilinx</h1>
```
Comment (anthonydillon) [inline templates/download/amd-xilinx/index.html:16]:
```suggestion
      <h1>Install Ubuntu on <br class="u-hide--small">AMD-Xilinx</h1>
```

File: templates/download/amd-xilinx/index.html [hunk 2]
```diff
@@ -2,19 +2,19 @@
...
+      <p>Run Ubuntu on your AMD-Xilinx Zynq UltraScale+ MPSoC-based evaluation boards and Kria SOMs. </p>
       <p>Pick the OS image to match your hardware, flash it onto SD/microSD card, load it onto your board and away you go.</p>
```
Comment (anthonydillon) [inline templates/download/amd-xilinx/index.html:18]:
```suggestion
      <p>Pick the OS image to match your hardware, flash it onto SD/microSD card, load it onto your board and away you go.</p>
      <p>All images are 64-bit builds of Ubuntu Desktop.</p>
```

**Review by anthonydillon — COMMENTED** [review]
There are a few comments from @juanruitina in the copy doc which might be worth addressing first.

File: templates/download/amd-xilinx/tab-3.html [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+    <table aria-label="AMD-Xilinx Ubuntu Software Development Kit table">
+      <thead>
+        <tr>
+          <th>Image name</th>
+          <th colspan="2">Description</th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr>
+          <th>
```
Comment (anthonydillon) [inline templates/download/amd-xilinx/tab-3.html:53]: This should be a `<td>`

File: templates/download/amd-xilinx/tab-3.html [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+    <h4>First time installing Ubuntu on AMD-Xilinx?</h4>
+    <p>Please see the <a href="https://xilinx-wiki.atlassian.net/wiki/spaces/A/pages/2363129857/Getting+Started+with+Certified+Ubuntu+22.04+LTS+for+Xilinx+Devices">Getting Started with Certified Ubuntu 22.04 LTS for AMD-Xilinx Devices</a> for more information.</p>
+    <p><strong>AMD-Xilinx Ubuntu Software Development Kit</strong></p>
+    <p>Ubuntu developers targeting the Versal&trade; Evaluation Kits may also want to download the following files:</p>
+    
+    <table aria-label="AMD-Xilinx Ubuntu Software Development Kit table">
+      <thead>
+        <tr>
+          <th>Image name</th>
+          <th colspan="2">Description</th>
```
Comment (anthonydillon) [inline templates/download/amd-xilinx/tab-3.html:48]: `colspan` is to merge cells and in this case, there are no cells to merge into causing invalid HTML. I would suggest removing the col spans as it looks nice without them. If you want a different width you can add `width="65%"` to the table header and all child cells in that column will inherit it.

Comment (anthonydillon) [reply templates/download/amd-xilinx/tab-3.html:48]: The width doesn't look great on small screens: 
image

**Review by anthonydillon — APPROVED** [review]
LGTM a few comments in the code but mostly trivial

**Comment by mtruj013** [comment]
Thanks @anthonydillon ! @juanruitina's comments are to be addressed in a future revamp at the request of the pdm. Full context [here](https://warthogs.atlassian.net/browse/WD-3924)

**Pull request merged**